handlers/common.go

```diff
@@ -278,16 +278,14 @@ func createExtractedConfig(event *events.Event, machine *client.Machine) (string
 	tarfileWriter := tar.NewWriter(fileWriter)
 	defer tarfileWriter.Close()
 
-	if err := addDirToArchive(baseDir, tarfileWriter); err != nil {
+	if err := addDirToArchive(baseDir, machine.Name, tarfileWriter); err != nil {
```

Comment:
The `machine.Name` needs to be sanitized because it is coming from the user.  Should probably only allow `[a-zA-Z0-9-.]` and then replace the other characters with a `-` or `_`.